compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -4610,6 +4610,7 @@ private List<BVarSymbol> defineParameters(List<BLangSimpleVariable> params, Symb
                 }
             }
             if (varNode.flagSet.contains(Flag.INCLUDED)) {
+                requiredParamNames.add(symbol.name.value);
```

Comment:
We need to add this as a breaking change.

Reply:
Create PR(https://github.com/ballerina-platform/ballerina-dev-website/pull/6559) for this